### PR TITLE
Madaoo/my article about lists in python

### PR DESCRIPTION
**Pull request by MadaooQuake**

### CODE REVIEW

File: list/README.md [hunk 1]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
 
 ## Merge lists
 
+Python can merge lists with each other in a variety of ways, in this section we will go through the most popular merged lists and which theses are offered by the latest version of Python.
```
Comment (amyska) [inline list/README.md]: Python can merge lists with each other in a variety of ways. In this section we will go through the most popular ways to 
 merge lists and which are offered by the latest version of Python.

File: list/README.md [hunk 2]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
...
+
+The first way, which is simple, but unfortunately not the best solution.
```
Comment (amyska) [inline list/README.md]: dlaczego nie jest best solution by sie przydalo :)

File: list/README.md [hunk 3]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
...
+
+```
+list1 = [1,2,3,4]
+list2 = [5,6,7,8]
+
+merged_list = list1 + list2
+print(merged_list)
+```
+Example 1 output :
```
Comment (amyska) [inline list/README.md]: Example 1 output:

bez spacji :)

File: list/README.md [hunk 4]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
...
+['beaver', 'beard']
+```
+
 ## List Comprehensions
 
 ## The Map() Function
 
 ## The Zip() functions
 
+The Zip() function in Python serves to create Tuple object. The Zip() function as paremteter use Lists, strings, numbers, tuple, etc.
```
Comment (amyska) [inline list/README.md]: as paremeter

File: list/README.md [hunk 5]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
...
+
 ## List Comprehensions
 
 ## The Map() Function
 
 ## The Zip() functions
 
+The Zip() function in Python serves to create Tuple object. The Zip() function as paremteter use Lists, strings, numbers, tuple, etc. 
+The Zip() function returns an iterator of tuples, where the i-th tuple contains the i-th element from each of the argument sequences or iterables.
+The following example, using the zip(0) function, creates a tuple object from two lists:
```
Comment (amyska) [inline list/README.md]: zip(0) ? skad to zero :)

File: list/README.md [hunk 6]
```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
...
+```
+list1 = {"one", "two", "three", "four"}
```
Comment (amyska) [inline list/README.md]: it is set not list :)

File: list/README.md (hunk 2 above)
Comment (MadaooQuake) [reply list/README.md]: To zostawiam czytającemu

File: list/README.md (hunk 5 above)
Comment (MadaooQuake) [reply list/README.md]: nie miało byc zero, za szybko pisalem :+1:

**Pull request merged**